### PR TITLE
Adding docs for new component GOCR

### PR DESCRIPTION
**Pull request by grea09**
**Description:**

Documentation for a new component for GOCR. It allows user to extract text simply from an image and even use reinforcement learning to fit their needs and improve accuracy.

I don't know how to add the logo:

logo

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#20871

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/image_processing.gocr.markdown [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  required: false
+  default: false
+  type: boolean
+unrecognized:
+  description: String to output for any unrecognized characters.
+  required: false
+  default: '_'
+  type: string
+extra_arguments:
+  description: Other arguments to use. Like `-m 2`, `-p database`, etc. See [`manpage`](https://linux.die.net/man/1/gocr).
```
Comment (klaasnicolaas) [inline source/_components/image_processing.gocr.markdown]:
```suggestion
  description: "Other arguments to use. Like `-m 2`, `-p database`, etc. See [`manpage`](https://linux.die.net/man/1/gocr)."
```

File: source/_components/image_processing.gocr.markdown [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  required: false
+  default: 0
+  type: integer
+chars:
+  description: String containing all accepted characters to output as result of the recognition.
+  required: false
+  default: ''
+  type: string
+negate:
+  description: `True` if you want the image to be inverted before recognition. The process works better if the text is black on white.
```
Comment (klaasnicolaas) [inline source/_components/image_processing.gocr.markdown]:
```suggestion
  description: "`True` if you want the image to be inverted before recognition. The process works better if the text is black on white."
```

File: source/_components/image_processing.gocr.markdown [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+  required: false
+  default: false
+  type: boolean
+unrecognized:
+  description: String to output for any unrecognized characters.
+  required: false
+  default: '_'
```
Comment (klaasnicolaas) [inline source/_components/image_processing.gocr.markdown]:
```suggestion
  default: "'_'"
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by klaasnicolaas** [comment]
Parent PR still open

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed.

**Pull request closed without merging**